src/Application/Common/Models/PaginationQuery.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace CleanArchitecture.Application.Common.Models
+{
+    public class PaginationQuery
+    {
+        public int PageNumber { get; set; } = 1;
+        public int PageSize { get; set; } = 10;
+    }
+}
```

Comment:
Recommend dropping pagination query, I'd rather just add the properties directly to the query. That way query can remain isolated from one another and they remain explicit with regards to the input requirements.